scripts/download_and_archive.sh

```diff
@@ -1,103 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
I'd really prefer that you keep using Bash... I don't think we have any advantage of using POSIX.

Reply:
No Problem